README.md

```diff
@@ -116,3 +116,32 @@ kubectl create secret generic fluentbit-log --from-literal azure_log_workspace_i
 
 kubeseal --format=yaml --cert=k8s/<env>/pub-cert.pem < /tmp/fluentbit-log.json > k8s/<env>/common/neuvector/fluentbit-log.yaml
 ```
+### Traefik
+
+In case you are enforcing ssl on Traefik( Refer Demo for flux config)  with an existing pfx in keyvault ,extract the certificate and key using below: 
```

Comment:
```suggestion
In case you are enforcing ssl on Traefik( Refer Demo for flux config) with an existing pfx in keyvault, extract the certificate and key using below: 
```